test/acquisition/test_proximal.py

```diff
@@ -56,11 +58,25 @@ def test_proximal(self):
                 # test with and without transformed weights
                 for transformed_weighting in [True, False]:
                     # test with single outcome model
-                    model = (
-                        SingleTaskGP(train_X, train_Y, input_transform=input_transform)
-                        .to(device=self.device, dtype=dtype)
-                        .eval()
-                    )
+                    with warnings.catch_warnings():
+                        warnings.filterwarnings(
+                            "ignore",
+                            message=_get_single_precision_warning(torch.float32),
+                        )
+                        warnings.filterwarnings(
+                            "ignore",
+                            message=(
+                                "Input data is not contained to the unit cube. "
+                                "Please consider min-max scaling the input "
+                                "data.",
+                            ),
+                        )
```

Comment:
Could we instead of filtering the warnings just standardize the data (for the inputs we can do it manually when not using a `Normalize` transform)?

Reply:
For this one, yeah. I think there were some tests where it looked like the data was intentionally not standardized.

Reply:
Yep. It turns out that the single-precision warning is already suppressed by default by `BotorchTestCase.setUp`, but that didn't happen here since `setUp` was overridden. So I added a call to `super().setUp()`.